### DISCUSSION TITLE
How do Audit logs work?

### DISCUSSION THREAD

**Discussion by tluyben** [Budibase/budibase#3190] [Help]
I see couchdb is used to store audit logs but cannot find in the docs (or discussions) what they are or what they are used for or where they can be found?

**Comment by mjashanks** [comment]
Ah - you're on the ball :)

Yes, we have started recording an audit trail for some things, but only in the background. There is no UI for this yet. You can consider this to **not** be a feature yet even though we've started it!

**Reply by gauraveauk** [reply]
Did you create a UI for the Audit trail.Wanted to check if this is something supported now in budibase.

**Reply by MentalGear** [reply]
2nd that ! Any user activity log?

**Reply by shogunpurple** [reply]
Nothing yet - although this work is planned for Q4. This feature will be part of the budibase paid plans
